test/schema-serialization.test.js

```diff
@@ -95,17 +98,40 @@ test('Different content types', t => {
           }
         },
         201: {
-          content: { type: 'string' }
+          content: {
+            '*/*': {
+              schema: { type: 'string' }
+            }
+          }
```

Comment:
I found a bug when the `keys` here is exactly one, it is actually being transformed as media-type of `type` which accept `{ type: 'object', properties: { type: 'string' } }`.